recipe-server/client/control_new/components/common/FormActions.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+
+/**
+ * Container for buttons displayed at the bottom of a form.
+ *
+ * Example:
+ *
+ * <FormActions>
+ *   <FormActions.Primary>
+ *     <Button>Save</Button>
+ *   </FormActions.Primary>
+ *   <FormActions.Secondary>
+ *     <Button>Delete</Button>
+ *   </FormActions.Secondary>
+ * </FormActions>
+ */
+export default function FormActions({ children }) {
```

Comment:
I'm not sure this component is necessary at all. Here, it's basically is a div wrapper, whereas in control v1 it handles a bit more data to automatically show/hide necessary buttons. Is that the plan here?